packages/next/build/utils.ts

```diff
@@ -209,35 +206,18 @@ export async function getCacheIdentifier({
     .digest('hex')
 }
 
-export async function getPageInfo(
+export async function getPageSizeInKb(
```

Comment:
It might be good to keep this as `getPageInfo` since we will probably need to gather more info for pages in the future, for example, I'm also using it on the auto static/dynamic branch

Reply:
Considering a lot of the function changed it'd probably be best to create a new function for whatever other use-case it needs to serve IMO.